MonkeyChat/ChatViewController.swift

```diff
@@ -787,7 +821,45 @@ extension ChatViewController {
     try! AVAudioSession.sharedInstance().setActive(false, with: AVAudioSessionSetActiveOptions.notifyOthersOnDeactivation)
     AudioServicesPlayAlertSound(kSystemSoundID_Vibrate)
   }
+}
+
+// MARK: - Audio Recording delegate
+extension ChatViewController: RGCircularSliderDelegate {
+  func audioDidBeginPlaying(_ audioSlider: Any?) -> Bool {
+    try! AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayAndRecord, with: .defaultToSpeaker)
+    if (self.audioBubbleOnPlay != nil){
```

Comment:
Is it necessary that this delegate method return true ? can we change the protocol?